libclc/CMakeLists.txt

```diff
@@ -34,11 +34,6 @@ set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS
   clc/lib/generic/SOURCES;
   # libspirv libraries
   libspirv/lib/generic/SOURCES;
-  libspirv/lib/amdgcn-amdhsa/SOURCES;
```

Comment:
We shouldn't be removing these  lines, since we're still using them. Removing them here means we won't reconfigure when one of the `SOURCES` files gets changed, and so we'd end up with weird incomplete builds.